hooks/use-is-plugin-active/index.ts

```diff
@@ -0,0 +1,29 @@
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import type { Plugin } from '@wordpress/core-data';
+
+const ACTIVE_STATUSES = ['active', 'network-active'] as const;
```

Comment:
@fabiankaegy just wanted to confirm if `network-active` is a valid value.

It is not in the list here:

https://github.com/WordPress/gutenberg/blob/8c77df8c1052015f26b57d65fe5f043cd4f774ac/packages/core-data/src/entity-types/plugin.ts#L76

Reply:
I found out about this the hard way on a project when it didn't work. I was also surprised to find it but apparently it exists.